packages/lib-utils/src/app/AppInitSDK.tsx

```diff
@@ -5,14 +5,15 @@ import * as React from 'react';
 import { Provider } from 'react-redux';
 import type { UtilsConfig } from '../config';
 import { isUtilsConfigSet, setUtilsConfig } from '../config';
-import type { InitAPIDiscovery } from '../types/api-discovery';
+import type { APIResourceList, InitAPIDiscovery } from '../types/api-discovery';
 import { initAPIDiscovery } from './api-discovery';
 import { useReduxStore } from './redux';
 
 type AppInitSDKProps = {
   configurations: {
     apiDiscovery?: InitAPIDiscovery;
     apiPriorityList?: string[];
+    staticApiModels?: Record<string, APIResourceList>;
```

Comment:
Nit: should we capitalize `API`? Not blocking.

Reply:
Good point!